sycl/test-e2e/DeviceLib/built-ins/offload-prec-div-sqrt.cpp

```diff
@@ -0,0 +1,61 @@
+// RUN: %{build} -foffload-fp32-prec-div -foffload-fp32-prec-sqrt -o %t.out
+// RUN: %{run} %t.out
+
+// Test if div and sqrt become precise from IEEE-754 perspective when
+// -foffload-fp32-prec-div -foffload-fp32-prec-sqrt are passed.
+
+#include <cmath>
+#include <sycl/detail/core.hpp>
+#include <sycl/usm.hpp>
+
+constexpr float value = 560.0f;
+constexpr float divider = 280.0f;
```

Comment:
This test of division is `560/280` ? That has an exact solution ( 2 ) .  I know the float number space is sparse, but is 560 / 280 really a test that'll result in a measurable ULP difference?  Wouldn't  279 be a better divider?

Reply:
Changed the divider to 279.9f